### ISSUE TITLE
Marketplace model OrientedMagneticChains bugs

### ISSUE THREAD

**Issue by astellhorn** [SasView/sasview#2775]
**Describe the bug**
Functionality test and review of the model "OrientedMagneticChains" from the marketplace (https://marketplace.sasview.org/models/143/ ):
1) the "ViewingAngle" parameter seems defined twice, which gives an error when loading the model in sasview 5.0.6. It is defined (i) as dependent parameter in the Iqxy(…, ViewingAngle,…) function in line 135 of the .c-file, and (ii) in line 142 of the .c-file again as double ViewingAngle = atan(y/x)*180.0/PiNum. It is unclear to me which one should be the correct one to use. If deleted from line 142, then the model can be loaded in sasview, but changing the inserted viewingangle does not change the scattering pattern.
2) Not all parameters are well defined. A table of parameters and what they do would be good

**To Reproduce**
1. Go to (https://marketplace.sasview.org/models/143/)
2. Download the .py and .c files of the OrientedMagneticChains model
3. Save into the models folder of sasview version 5.0.6
4. Load model under "spheres" "OrientedMagneticChains" in the fitting page

**SasView version**
 - Version: 5.0.6

**Operating system (please complete the following information):**
 - MacOS

**Comment by butlerpd** [comment]
Indeed, this does not seem to be a model that has been submitted to sasmodels yet? @astellhorn, @dehoni or @JoGaudet , is this one of those submitted in the PR already? If not, do you plan to include this model in the future?  In general marketplace models should be updated by their authors. However, the process to include them in the sasmodels distribution would require cleaning up the code and fixing any bugs.

If there is no plan to pull the model in we should close the issue IMO. If there is a plan, but not immediate, then we should create an issue in sasmodels to move that model into sasmodels and add the comments above as a known required fix... then close this issue

**Comment by dehoni** [comment]
This is a model on the marketplace [https://marketplace.sasview.org/models/143/](url). If some documentation for this model can be found, it would already give certainty on the parameters.
